src/renderer/mainWindow/map/MapContainer.tsx

```diff
@@ -318,7 +319,7 @@ export default class MapContainer extends Component<ThemeProps, State> {
     boundingBoxes.forEach((boundingBox): void => {
       if (!newBoundingBoxes[boundingBox.name]) {
         newBoundingBoxes[boundingBox.name] = {
-          color: boundingBox.color || 'red',
+          color: boundingBox.color || 'blue',
```

Comment:
pls keep default to red, or perhaps change it to a more blank color (like white)